source/extensions/filters/http/composite/config.cc

```diff
@@ -10,13 +10,14 @@ namespace Extensions {
 namespace HttpFilters {
 namespace Composite {
 
-Http::FilterFactoryCb CompositeFilterFactory::createFilterFactoryFromProtoTyped(
+absl::StatusOr<Http::FilterFactoryCb> CompositeFilterFactory::createFilterFactoryFromProtoTyped(
     const envoy::extensions::filters::http::composite::v3::Composite&,
-    const std::string& stat_prefix, Server::Configuration::FactoryContext& factory_context) {
+    const std::string& stat_prefix, DualInfo dual_info,
+    Server::Configuration::ServerFactoryContext&) {
```

Comment:
I think it's wrong for now. Because we may need to use message validation visitor of FactoryContext.

But anyway, there are lots of similar problems in our code base. 🤣 Let's to resolve them in future by one punch.

Reply:
This method itself does not access message validator.

The callers do access it:
1) downstream: https://github.com/envoyproxy/envoy/blob/d07b522e68d0f1c53e4e76d61445a892626ff285/source/extensions/filters/http/common/factory_base.h#L143
2) upstream: 
https://github.com/envoyproxy/envoy/blob/d07b522e68d0f1c53e4e76d61445a892626ff285/source/extensions/filters/http/common/factory_base.h#L154

Do you mean 2) is wrong?  BTW, this pattern indeed is used by all dual filters.